driver.py

```diff
@@ -351,7 +351,8 @@ def insert_row(table, row, checked=False):
     cmd += ")"
     try:
         pgSqlCur.execute(cmd)
-
+        if not checked:
+            pgSqlConn.commit()
```

Comment:
Why only commit the cursor if not checked?

Reply:
This was to force a commit when the put endpoint was used.  It doesn't use the process file function, which contains the commit, it goes straight the intake_row function.

Reply:
Yeah but why do you not call pgSqlConn.commit() unless checked=False?

Reply:
My thinking was since the process file calls the insert_row multiple times and then does a commit after all are inserted don't commit rows that are part of a larger set being inserted,  In case of failure during file insert, we can then assume no records from a file are inserted.    Would we rather have every row commited as its inserted no matter what?

Reply:
Okay cool, no that's good thinking, I just wasn't clear on what the checked flag was doing